packages/flutter/test/material/dropdown_menu_test.dart

```diff
@@ -3847,6 +3847,33 @@ void main() {
     },
     variant: TargetPlatformVariant.all(),
   );
+
+  // Regression test for https://github.com/flutter/flutter/pull/161903
```

Comment:
This seems the PR link instead of issue link? I think we don't need to add this line if it's just a new feature:)

Reply:
Thank you for review. Found the failure, it's due to the usage of minLines in doc macro.